com.dynamo.cr/com.dynamo.cr.bob/src/com/dynamo/bob/Project.java

```diff
@@ -1189,6 +1195,163 @@ public void scanJavaClasses() throws IOException, CompileExceptionError {
         scan(scanner, "com.defold.extension.pipeline");
     }
 
+    private Future buildRemoteEngine(IProgress monitor, ExecutorService executor) {
+        Callable<Void> callable = new Callable<>() {
+            public Void call() throws Exception {
+                logInfo("Build Remote Engine...");
+                TimeProfiler.start("Build Remote Engine");
```

Comment:
I'm not sure about the timeprofiler start and stop here since this is now running in parallel with other profiled work.